src/test/kotlin/apply/domain/recruitmentitem/RecruitmentItemRepositoryTest.kt

```diff
@@ -0,0 +1,37 @@
+package apply.domain.recruitmentitem
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+
+@DataJpaTest
+internal class RecruitmentItemRepositoryTest(
+    @Autowired
```

Comment:
`@Autowired`를 제거하려면`@TestConstructor`를 사용할 수 있습니다.
https://docs.spring.io/spring/docs/current/spring-framework-reference/testing.html#integration-testing-annotations-testconstructor

```suggestion
@TestConstructor(autowireMode = AutowireMode.ALL)
@DataJpaTest
internal class RecruitmentItemRepositoryTest(
```